### DISCUSSION TITLE
can anyone help me with this?

### DISCUSSION THREAD

**Discussion by heyyshaarib** [LazyVim/LazyVim#4753] [Q&A]
![Screenshot 2024-11-10 082407](https://github.com/user-attachments/assets/e3eff1ff-2c42-4f03-9484-a08321e10cb3)

**Comment by folke** [comment]
I don't see how this can happen... Did you disable snacks.nvim?

**Comment by ada4a** [comment]
You probably updated (with "U"), but didn't install the new plugin (with "I")
